### PR TITLE
feat: add docs for the Card component

### PR DESCRIPTION
**Pull request by AnnaRybkina**
The docs are updated. Ported from 👇 
React: https://react.fabric-ds.io/card
Vue: https://vue.fabric-ds.io/card
Elements: https://elements.fabric-ds.io/card

http://localhost:5174/tech-docs/components/card/ - local tests

image

**NOTE** TORI colors needs some adjustments. Will be fixed in the tokens repo

### CODE REVIEW

File: docs/components/card/elements.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
+## Import
+
+The toggle component allows you to render checkboxes. All you have to do is specify `type="checkbox"` for the Toggle and let the component handle the rest. Toggle is built to handle both single and multiple options.
```
Comment (BalbinaK) [inline docs/components/card/elements.md]: wrong component :)

File: docs/components/card/elements.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+You must keep track of state yourself. The state has to be handled differently depending on whether you want to use single or multiple options. See examples below.
+
+
+```js
+import { wCard } from '@warp-ds/elements';
+```
```
Comment (BalbinaK) [inline docs/components/card/elements.md]: We don't need to import Warp elements this way 😊  You can check e.g. pages for Breadcrumbs and Box for how elements docs were set up. I see Buttons also have a redundant import bit in the elements docs.

File: docs/components/card/react.md [hunk 3]
```diff
@@ -0,0 +1,356 @@
+## Import
+
+The toggle component allows you to render checkboxes. All you have to do is specify `type="checkbox"` for the Toggle and let the component handle the rest. Toggle is built to handle both single and multiple options.
```
Comment (BalbinaK) [inline docs/components/card/react.md]: same here :)

**Review by BalbinaK — COMMENTED** [review]
💪 Just a few comments:

File: docs/components/card/elements.md (hunk 1 above)
Comment (AnnaRybkina) [reply docs/components/card/elements.md]: 🤦 https://github.com/warp-ds/tech-docs/pull/39/commits/01519cb79fc1a1d0a8fb8272376cf7743637daf0

File: docs/components/card/elements.md (hunk 2 above)
Comment (AnnaRybkina) [reply docs/components/card/elements.md]: Fixed https://github.com/warp-ds/tech-docs/pull/39/commits/01519cb79fc1a1d0a8fb8272376cf7743637daf0

File: docs/components/card/react.md (hunk 3 above)
Comment (AnnaRybkina) [reply docs/components/card/react.md]: 🤦 https://github.com/warp-ds/tech-docs/pull/39/commits/01519cb79fc1a1d0a8fb8272376cf7743637daf0

**Review by BalbinaK — APPROVED** [review]

**Pull request merged**